Userland/Utilities/js.cpp

```diff
@@ -417,6 +417,7 @@ JS_DEFINE_NATIVE_FUNCTION(ReplObject::save_to_file)
 
 JS_DEFINE_NATIVE_FUNCTION(ReplObject::exit_interpreter)
 {
+    s_editor->save_history(s_history_path.to_deprecated_string());
```

Comment:
(it looks like it could be extracted to a separate function–if possible and not an issue)